.github/workflows/continuous-integration.yaml

```diff
@@ -352,9 +362,10 @@ jobs:
           sed -i 's|^deb http://archive|deb http://fr.archive|g' /etc/apt/sources.list
           apt update && apt install git git-lfs -y
 
-      # By default, `git clone` downloads all LFS files, which we want to avoid in regular CI
+      # By default, `git clone` downloads all LFS files, which we want to avoid in CIs other than
+      # weekly ones (which also test notebooks)
       - name: Disable LFS download by default
-        if: ${{ !fromJSON(env.IS_WEEKLY) && !fromJSON(env.IS_RELEASE) }}
```

Comment:
no need to download all LFS files in release CIs since we only need tests files like regular CIs